x-pack/legacy/plugins/siem/public/components/open_timeline/helpers.ts

```diff
@@ -97,6 +105,32 @@ export const defaultTimelineToTimelineModel = (
           return acc;
         }, {})
       : {},
+    filters:
+      timeline.filters != null
+        ? timeline.filters.map(filter => ({
+            $state: {
+              store: 'appState',
+            },
+            meta: {
+              ...filter.meta,
+              ...(filter.meta && filter.meta.field != null
+                ? { params: parseString(filter.meta.field) }
+                : {}),
+              ...(filter.meta && filter.meta.params != null
+                ? { params: parseString(filter.meta.params) }
+                : {}),
+              ...(filter.meta && filter.meta.value != null
+                ? { value: parseString(filter.meta.value) }
+                : {}),
+            },
+            ...(filter.exists != null ? { exists: parseString(filter.exists) } : {}),
```

Comment:
I think that here we do not need it here because it is coming back from our timeline saved object and if it is different to null we need to parse the string as simple as that. Since we are using the type guard for persistence.